indico/modules/events/editing/client/js/components/FileManager/index.jsx

```diff
@@ -91,6 +93,13 @@ function FileType({uploadURL, fileType, uploads}) {
   return (
     <div styleName="file-type">
       <h3>{fileType.name}</h3>
+      <TooltipIfTruncated customText={fileType.extensions.map(ext => ext).join(', ')}>
```

Comment:
`.map(ext => ext)` doesn't do anything

Reply:
Oops, it was leftover from previous implementations. Thanks for pointing out